File: carsharing-api/carsharing-api/Database/Models/Car.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace carsharing_api.Database.Models;

public class Car
{
    [Column("id")]
    public int Id { get; set; }
    [Column("brand")]
    public string Brand { get; set; }
    [Column("model")]
    public string Model { get; set; }
    [Column("image_path")]
    public string ImagePath { get; set; }
    [Column("fuel_level")]
    public string FuelLevel { get; set; }
    [Column("car_plate")]
    public string CarPlate { get; set; }
    [Column("class")]
    public int Category { get; set; }
    [Column("params")]
    public string Params { get; set; }
    [Column("insurance")]
    public int Insurance { get; set; }
    [Column("rate")]
    public int Rate { get; set; }
    [Column("address")]
    public string Address { get; set; }
    [Column("reservation")]
    public int? Reservation { get; set; }
}

File: carsharing-app/app/src/main/java/com/syndicate/carsharing/database/models/Car.kt
package com.syndicate.carsharing.database.models

import com.fasterxml.jackson.annotation.JsonProperty

data class Car (
    @JsonProperty("id") val id: Int?,
    @JsonProperty("brand") val brand: String?,
    @JsonProperty("model") val model: String?,
    @JsonProperty("image_path") val imagePath: String?,
    @JsonProperty("fuel_level") val fuelLevel: String?,
    @JsonProperty("car_plate") val carPlate: String?,
    @JsonProperty("class") val category: Int?,
    @JsonProperty("params") val params: String?,
    @JsonProperty("insurance") val insurance: Int?,
    @JsonProperty("rate") val rate: Int?,
    @JsonProperty("address") val address: String?,
    @JsonProperty("reservation") val reservation: Int?
)